addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -186,7 +202,11 @@ def action_print(self):
 
     def action_set_quantities_to_reservation(self):
         self.ensure_one()
-        self.picking_ids.filtered("show_validate").action_set_quantities_to_reservation()
+        self.picking_ids.filtered("show_set_qty_button").action_set_quantities_to_reservation()
+
+    def action_clear_quantities_to_zero(self):
+        self.ensure_one()
```

Comment:
Not a big deal, but the `ensure_one` is useless here as this method could be called on multiple batch with no problem (same for `action_set_quantities_to_reservation`).